src/controllers/expenses.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const expensesServices = require('../services/expenses');
+const usersServices = require('../services/users');
+
+function getAll(req, res) {
+  const { userId, category, from, to } = req.query;
+
+  const searchParams = {
+    userId,
+    category,
+    from,
+    to,
+  };
+
+  const expenses = expensesServices.getAll(searchParams);
+
+  res.statusCode = 200;
+  res.send(expenses);
+}
+
+function addExpense(req, res) {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  const user = usersServices.getById(userId);
+
+  if (!user) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const expenseData = {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  };
```

Comment:
We should validate the request body. What if I don't send the required fields?
<img width="275" alt="image" src="https://user-images.githubusercontent.com/32794096/219646231-5b344e60-f26a-45fd-96e5-8691c4099415.png">